test/test_benchmark.py

```diff
@@ -240,6 +269,21 @@ def get_output_files(self):
     return ret
 
 
+# This benchmarker will make a test benchmark build with Emscripten and record
+# the file output sizes in out/test/stats.json. The file format is specified at
+# https://skia.googlesource.com/buildbot/+/refs/heads/main/perf/FORMAT.md
+# Running the benchmark will be skipped.
+class SizeBenchmarker(EmscriptenBenchmarker):
+  record_stats = True
+
+  def __init__(self, name):
+    # do not set an engine, as we will not run the code
+    super().__init__(name, engine=None)
+
+  def run(self, args):
+    return
```

Comment:
Can this
```suggestion
  # we will not actually run the benchmarks
  run = None
```
work together with the next comment?